Corvus.Extensions.Samples/README.md

```diff
@@ -0,0 +1,13 @@
+# Corvus.Extensions sample notebooks
+
+In this folder, you will find a number of notebooks containing examples for the extension methods in the [`Corvus.Extensions\Solutions\Corvus\Extensions`](https://github.com/corvus-dotnet/Corvus.Extensions/tree/samples-readme/Solutions/Corvus.Extensions/Corvus/Extensions) folder.
+
+## How to use .NET Interactive notebooks
+
+In order to use .NET Interactive notebooks, download the .NET Interactive Notebooks extensions from the [marketplace](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode). 
+
+You'll need to have the following pre-requisites installed:
+ - Visual Studio Code (can be downloaded from [here](https://code.visualstudio.com/))
+ - Install the latest [.NET 6 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
```

Comment:
The word "Install" is redundant here as this is a list of things that need to be installed.

Reply:
Removed the word "install"